src/internal/internalAPIs.ts

```diff
@@ -115,6 +115,20 @@ export function ensureInitialized(...expectedFrameContexts: string[]): void {
   }
 }
 
+/**
+ * Checks whether the platform has knowledge of this API by doing a comparison
+ * on API required version and platform supported version of the SDK
+ * @param requiredVersion SDK version required by the API
+ */
+export function IsAPISupportedByPlatform(requiredVersion: string = defaultSDKVersionForCompatCheck): boolean {
+  if (!GlobalVars.initializeCalled) {
+    throw new Error('The library has not yet been initialized');
```

Comment:
actually this piece of code was copied from ensureInitialized method, to make sure this method gets called after initialize is called. On second thoughts, the check seemed redundant (because ensureInitialize throws same error and we anywaysa always call that method before isAPISupported method), so removed it.